src/apps/backend/modules/account/account_service.py

```diff
@@ -24,6 +26,8 @@ def get_account_by_username_password(*, params: AccountSearchParams) -> Account:
     )
     return Account(
       id=str(account.id),
+      first_name=account.first_name,
       hashed_password=account.hashed_password,
```

Comment:
I understand this is not related to this PR but`hashed_password` should not be populated on Account. Can we please remove it?

Reply:
Removed the `hashed_password` from Account class